storage/innobase/include/ut0ut.h

```diff
@@ -305,6 +305,33 @@ operator<<(
 	return(lhs);
 }
 
+/** This is a wrapper class, used to print any number in IEC style */
+struct bytes_iec {
+ explicit bytes_iec(unsigned long long t): m_val(t) {}
+ const unsigned long long m_val;
+};
```

Comment:
Indent with 2 spaces, not 1.
I would omit a `m_` prefix for data members that are directly accessible.